h2o-core/src/main/java/hex/Model.java

```diff
@@ -1544,10 +1569,11 @@ protected AdaptFrameParameters makeAdaptFrameParameters() {
     String getOffsetColumn();
     String getFoldColumn();
     String getResponseColumn();
+    String getTreatmentColumn();
     double missingColumnsType();
     int getMaxCategoricalLevels();
     default String[] getNonPredictors() {
-      return Arrays.stream(new String[]{getWeightsColumn(), getOffsetColumn(), getFoldColumn(), getResponseColumn()})
+      return Arrays.stream(new String[]{getWeightsColumn(), getOffsetColumn(), getFoldColumn(), getResponseColumn(), getTreatmentColumn()})
```

Comment:
this could be simplified using `Stream.of`

Reply:
Good point, but no refactor in this code is allowed within this PR.